lib/libraries/cms/html/html.php

```diff
@@ -536,7 +536,7 @@ protected static function includeRelativeFiles($folder, $file, $relative, $detec
 					 */
 					foreach ($files as $file)
 					{
-						$path = JPATH_ROOT . "/$file";
+						$path = JPATH_WEB . "/$file";
```

Comment:
@allanpilarca If you make changes like this please keep a log of the exact files that are changed and add the changes here: https://github.com/joomlatools/joomlatools-platform/wiki/Upstream-changes as you can see I have included the commits and the lines of code that have been changed.